RoomLocator/RoomLocator.Api/Controllers/ScadadataController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using RoomLocator.Data.Services;
+using RoomLocator.Domain.ViewModels;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Newtonsoft.Json;
+
+
+namespace RoomLocator.Api.Controllers
+{
+    [ApiVersion("1.0")]
+    [Route("api/v{version:apiVersion}/[controller]")]
+    [ApiController]
+    public class ScadadataController :Controller
```

Comment:
Jeg ville nok nærmere kalde den `StatusController` eller noget i den stil. I vores originale planer talte vi om at systemet potentielt kunne dække hele DTU. Scandata API'et er noget biblioteket bruger. De bruger også modcam. Principielt set var det meningen at vores API skulle bruge data fra både Scandata samt Modcam for at kunne give et mere præcist billede af hvordan det er lige nu på biblioteket. Det kommer ikke til at være aktuelt nu, det er fint at starte simpelt med bare Scandata API'et, men det er mere for at fortælle hvorfor navnet kan være misvisende.

Hvis vi tager udgangspunkt i at systemet skal være i stand til at håndtere flere lokationer end blot biblioteket, så ville man for at få status fra biblioteket ramme endpointet `/api/v1/status/library`